CHANGELOG.md

```diff
@@ -13,3 +13,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/).
   of a keycloak implementation, the `realm` should be part of the `host`.
   This change is not backwards compatible! Just remove the `realm` property
   from your configuration and add it directly to the `host` property.
+- Add config option `userinfoEndpoint`, `scope` and `expiresIn`
```

Comment:
For users upgrading: when userinfo endpoint is not configured correctly this addon won't work correctly.

Hence I would split the optional config option and the not optional config option into two different lines and add a remark that userinfoEndpoints needs to be configured correctly otherwise it won't work.

Reply:
Updated the CHANGELOG according to the new changes.